modules/seqexec/server/src/main/scala/seqexec/server/tcs/Gaos.scala

```diff
@@ -46,29 +46,27 @@ object Gaos {
   case object OiOn extends ResumeCondition
   // PI will be turn off
   case object P1On extends ResumeCondition
-  // GAOS star guide will be turn off
-  case object GaosStarOn extends ResumeCondition
+  // Guided step
+  case object GaosGuideOn extends ResumeCondition
```

Comment:
I renamed it because in LGS+OI and LGS+P1 there is no guide star, so the old name didn't make sense.